package.json

```diff
@@ -17,14 +17,17 @@
     "url": "git://github.com/cheeriojs/cheerio.git"
   },
   "main": "./index.js",
+  "types": "types/index.d.ts",
   "files": [
     "index.js",
+    "types/index.d.ts",
     "lib"
   ],
   "engines": {
     "node": ">= 0.6"
   },
   "dependencies": {
+    "@types/node": "^14.11.2",
```

Comment:
adding TS things to deps is a *very bad idea* since the vast majority of consumers are not using TS.